### ISSUE TITLE
Access sample weights in custom metrics function

### ISSUE THREAD

**Issue by dickreuter** [keras-team/keras#9346]
Is there any way to access the sample weights from a custom metrics function, so I can use them for a custom built metric (i.e multiply them with y_true)?

**Comment by harshini-gadige** [comment]
Hi

If you think this is a feature request/enhancement, please open the issue with clear information on what is expected or required.

To ask questions, please see the following resources :

https://keras.io/
https://keras.io/#support
https://gitter.im/Keras-io/Lobby

Thanks!

If you think I made a mistake, please update the issue. We will reopen it.
